File: Php/poo/figure/library/square.php
<?php
    
        class square extends figureGeometric{
            private $side;
            private $calculate;
        
        public function __construct($side){
            parent::__construct("Square");
            $this->side=$side;
        }

        public function getSide(){
            return $this->side;
        }

        public function calculateArea(): float {
            return $this->side * $this->side;
            return $this->calculate;
        }
    }

    

File: Java/poo/figure/library/square.java
package library;

    public class square extends figureGeometric{
        private double side;
        

    public square(double side){
        super("Cuadrado");
        this.side = side;
    }

    public double getSide(){
        return this.side;

    }

        @Override
        public double calculateArea() {
            return this.side * this.side;
            
        }
    }